cpp/dolfin/io/xdmf_write.cpp

```diff
@@ -202,8 +202,8 @@ std::vector<std::int64_t> compute_topology_data(const mesh::Mesh& mesh,
                                                 int cell_dim)
 {
   // Create vector to store topology data
-  const int num_vertices_per_cell = mesh::num_cell_vertices(
-      mesh::cell_entity_type(mesh.cell_type(), cell_dim));
+  mesh::CellType cell_type = mesh::cell_entity_type(mesh.cell_type(), cell_dim);
```

Comment:
Cell type is also `const`.

Reply:
So it this rather an `enitity_cell_type`?

Reply:
Sure, i can rename